README.md

```diff
@@ -27,6 +27,11 @@ using RegressionFormulae, StatsModels, GLM, DataFrames
 
 `a / b` expands to `a + fulldummy(a) & b`.
 
+Numeric constants are special cased so that `/` so that performs division, making it possible to e.g. convert time to speed in the formula:
```

Comment:
```suggestion
Numeric constants are special cased so that `/` performs division, making it possible to e.g. convert time to speed in the formula:
```